wgpu-types/src/lib.rs

```diff
@@ -6165,3 +6160,26 @@ impl Default for InstanceDescriptor {
         }
     }
 }
+
+pub use send_sync::*;
+
+#[doc(hidden)]
```

Comment:
Done.

Reply:
Right, I forgot the context here.
The reason why I had `#[doc(hidden)]` here was because this is a public module.
See https://github.com/gfx-rs/wgpu/pull/3691#discussion_r1171754933.